projects/plugins/jetpack/modules/sharedaddy/sharing.php

```diff
@@ -121,24 +122,18 @@ public function process_requests() {
 	}
 
 	/**
-	 * Register Sharing settings menu page.
+	 * Register Sharing settings menu page in offline mode.
 	 */
 	public function subscription_menu() {
-		if ( ! defined( 'IS_WPCOM' ) || ! IS_WPCOM ) {
-			$active = Jetpack::get_active_modules();
-			if (
-				! in_array( 'publicize', $active, true )
-				&& ! current_user_can( 'manage_options' )
-			) {
-				return;
-			}
+		if ( ! ( new Status() )->is_offline_mode() ) {
```

Comment:
Given https://github.com/Automattic/jetpack/pull/35810#discussion_r1497440155 , I think that's probably enough of a check here 👍